sql-parser/src/test/java/io/crate/sql/IdentifiersTest.java

```diff
@@ -38,7 +38,7 @@ public void test_number_of_keywords() {
         // Either add the new term to `nonReserved` in `SqlBase.4g` or add a breaking changes entry and adapt this test.
         assertThat(
             (int) Identifiers.KEYWORDS.stream().filter(Identifiers.Keyword::isReserved).count(),
-            is(95)
+            is(96)
```

Comment:
> // Either add the new term to `nonReserved` in `SqlBase.4g` or add a breaking changes entry and adapt this test 

:)

Reply:
But CHECK is a reserved word, in PG:

```
create table t (
    id integer,
    qty integer constraint check check (qty > 0),
    primary key (id))


org.postgresql.util.PSQLException: ERROR: syntax error at or near "check"
  Position: 61
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
```

so adapt it is, but what does that mean?, is thus CHECK not a keyword susceptible of being included in the count? i.e. 95 -> 96 ?